language/diem-framework/modules/0L/Migrations.move

```diff
@@ -106,4 +106,33 @@ module MigrateTowerCounter {
   }
 
 }
+
+/// # Summary 
+/// Module to migrate the tower statistics from TowerState to TowerCounter
+module MigrateAutoPayBal {
+  use 0x1::AutoPay;
+  use 0x1::DiemAccount;
+  use 0x1::CoreAddresses;
+  use 0x1::Vector;
+  use 0x1::Migrations;
+
+  const UID:u64 = 2;
+  // Migration to migrate all wallets to be slow wallets
+  public fun do_it(vm: &signer) {
+    CoreAddresses::assert_diem_root(vm);
+    if (!Migrations::has_run(UID)) {
+      let enabled_accounts = AutoPay::get_enabled();
+      let i = 0;
+      let len = Vector::length<address>(&enabled_accounts);
+      while (i < len) {
+        let addr = Vector::borrow(&enabled_accounts, i);
+        let account_sig = DiemAccount::scary_wtf_create_signer(vm, *addr);
+        AutoPay::enable_autopay(&account_sig);
+      };
```

Comment:
Missing an increment of `i` here I think.

Reply:
Thanks. Sounds like something I would do :)